lrlex/src/lib/mod.rs

```diff
@@ -62,6 +65,9 @@ impl fmt::Display for LexBuildError {
             LexErrorKind::MissingSpace => "Rule is missing a space",
             LexErrorKind::InvalidName => "Invalid rule name",
             LexErrorKind::UnknownStartState => "Start state not known",
+            LexErrorKind::DuplicateStartState => "Start state already exists",
+            LexErrorKind::InvalidStartState => "Invalid start state",
+            LexErrorKind::InvalidStartStateName => "Invalid start state",
```

Comment:
Should this be `"Invalid start state name"`?

Reply:
Yes, I'll fix

Reply:
Fixed under https://github.com/softdevteam/grmtools/pull/318/commits/9bf38da33e6078106a1963382ecd6f09eb4cdd7a